tests/spec_tests/query_tests/group_by.yml

```diff
@@ -581,6 +538,22 @@ tests:
       - { "": { "a": { "$numberInt": "2" }, "gcount": { "$numberInt": "2" } } }
       - { "": { "a": null, "gcount": { "$numberInt": "0" } } }
 
+  - description: COUNT(*) correctness test unconditionally count rows
```

Comment:
Nice, thanks for adding these tests to show the COUNT(*) and COUNT(DISTINCT *) match behavior in terms of unconditionally counting.